benchmarks/cugraph/pytest-based/bench_cugraph_uniform_neighbor_sample.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import pytest
+import numpy as np
+import cupy as cp
+from cugraph.testing.mg_utils import start_dask_client, stop_dask_client
+import cudf
+import dask_cudf
+
+# If the rapids-pytest-benchmark plugin is installed, the "gpubenchmark"
+# fixture will be available automatically. Check that this fixture is available
+# by trying to import rapids_pytest_benchmark, and if that fails, set
+# "gpubenchmark" to the standard "benchmark" fixture provided by
+# pytest-benchmark.
+try:
+    import rapids_pytest_benchmark  # noqa: F401
+except ImportError:
+    import pytest_benchmark
+
+    gpubenchmark = pytest_benchmark.plugin.benchmark
+
+from cugraph import (
+    Graph,
+    uniform_neighbor_sample,
+)
+from cugraph.generators import rmat
+from cugraph.experimental import datasets
+from cugraph.dask import uniform_neighbor_sample as uniform_neighbor_sample_mg
+
+from cugraph_benchmarking import params
+
+_seed = 42
+
+
+def create_graph(graph_data):
+    """
+    Create a graph instance based on the data to be loaded/generated.
+    """
+    # FIXME: need to consider directed/undirected?
+    G = Graph(directed=True)
```

Comment:
I think this should this be a `cugraph.MultiGraph(directed=True)` , because there are multiple edges b/w same src and destination types.

Reply:
CC: @alexbarghi-nv for inputs on cugraph-pyg side